eng/Versions.props

```diff
@@ -7,8 +7,8 @@
     <MinorVersion>0</MinorVersion>
     <PatchVersion>6</PatchVersion>
     <SdkBandVersion>8.0.100</SdkBandVersion>
-    <PackageVersionNet7>7.0.$([MSBuild]::Add($(PatchVersion),14))</PackageVersionNet7>
-    <PackageVersionNet6>6.0.$([MSBuild]::Add($([System.Version]::Parse('$(PackageVersionNet7)').Build),11))</PackageVersionNet6>
+    <PackageVersionNet7>7.0.19</PackageVersionNet7>
+    <PackageVersionNet6>6.0.$([MSBuild]::Add($(PatchVersion),25))</PackageVersionNet6>
```

Comment:
Do you mind explaining the change?
- Why are we hardcoding 19 now?
- Why are we adding 6+25?

Reply:
> Do you mind explaining the change?
> 
> * Why are we hardcoding 19 now?

Because net7.0 is EOL the 14th so we don't want to compute the 7.0 version off the 8.0 version any more

> * Why are we adding 6+25?

The previous logic was 6.0.y = (8.0).x + (7.0).14 + 11  ∴ 8.0.6 ⇒ 6.0.31.  Now that 7.0 is pined I changed the logic to 6.0.y = 8.0.x + 25 ∴ 8.0.1 ⇒ 6.0.31

Reply:
And what did the 11 represent before?

Reply:
> And what did the 11 represent before?
just the difference between the 6 patch versiona and the 7 patch version.  We're exploring better ways to get the downlevel versions because this all rather fragile and annoying.

Reply:
And also cryptic for us non-wizards 😆. Thanks for the explanations.